### ISSUE TITLE
divide a tensor by the max value of this tensor in forward

### ISSUE THREAD

**Issue by achao2013** [apache/incubator-mxnet#4279]
i want to divide a tensor by the max value of this tensor.

Tensor<xpu,1,DType> m=aux_args[0].get<xpu, 1, DType>(s);
ASSIGN_DISPATCH(m,kWriteTo,(reduce_except_dim<1,mshadow::red::maximum(reshape(data,Shape2(1,data.shape_.Size())))));
Assign(out, req[activation::kOut],data*broadcast<1>(1/m,data.shape_));

but the compiler gives me an error,as following:

![image](https://cloud.githubusercontent.com/assets/5026576/21294499/fb225660-c578-11e6-8913-424250080584.png)

**Comment by piiswrong** [comment]
1/m -> Scalar<DType>(1)/m

**Comment by achao2013** [comment]
that's right,this is indeed a problem, but i am very confused,why in batch_norm-inl.h the  following code is right:
Assign(grad_in, req[batchnorm::kData], grad *
               broadcast<1>(
                   1.0f / F<mshadow_op::square_root>(moving_var + param_.eps), data.shape_)); @piiswrong

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
